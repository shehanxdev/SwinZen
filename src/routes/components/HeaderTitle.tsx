import { Text } from '@sz/components';
import { TextVariant } from '@sz/constants';

export function HeaderTitle({ children }) {
  return <Text variant={TextVariant.SubTitle2SemiBold}>{children}</Text>;
}
