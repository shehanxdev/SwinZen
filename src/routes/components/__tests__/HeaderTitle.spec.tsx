import { render } from '@testing-library/react-native';
import * as React from 'react';

import { HeaderTitle } from '../HeaderTitle';

describe('HeaderTitle', () => {
  const getRenderedComponent = () => render(<HeaderTitle children={'Title'} />);

  it(`should render correctly`, () => {
    expect(getRenderedComponent().toJSON()).toMatchSnapshot();
  });
});
