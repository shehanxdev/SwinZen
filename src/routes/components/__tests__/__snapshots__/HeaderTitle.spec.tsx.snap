// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderTitle should render correctly 1`] = `
<Text
  style={
    [
      {
        "color": "#F8F9FA",
        "lineHeight": 28,
      },
      {
        "fontFamily": "SourceSansPro-SemiBold",
        "fontSize": 24,
        "fontWeight": "600",
        "textAlign": "center",
      },
    ]
  }
>
  Title
</Text>
`;
